Shared/DataTransferObjects/AttendanceForCreationDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceForCreationDto(
+    Guid EmployeeId,
+    DateTime CheckInTime,
+    string Status,
```

Comment:
Consider renaming the `Status` property to `AttendanceStatus` to make its purpose clearer and avoid potential confusion with other status fields in broader models.

**Suggested change**
```suggestion
public record AttendanceForCreationDto(
    Guid EmployeeId,
    DateTime CheckInTime,
    string AttendanceStatus,
    string? Notes);
```